README.md

```diff
@@ -1,5 +1,5 @@
 # TapResearch-iOS-SDK
-TapResearch iOS SDK v2.5.8
+TapResearch iOS SDK v3.0.0-beta1
 
 
 For additional information, please see the [TapResearch iOS SDK integration guide](https://supply-docs.tapresearch.com/docs/ios-integration/).

```

Comment:
technicall, this will not be correct once the updated docs go out, but i think its fine for this release.